schema/markdownlint-config-schema-strict.json

```diff
@@ -1716,6 +1716,42 @@
       "type": "boolean",
       "default": true
     },
+    "MD059": {
+      "description": "MD059/descriptive-link-text : Link text should be descriptive : https://github.com/DavidAnson/markdownlint/blob/v0.37.3/doc/md059.md",
```

Comment:
Not sure what this should be set to, yet. Same for below.

Reply:
Whoops looks like I was not supposed to manually update it. I see a few potentially relevant scripts. Undoing!